src/integrationTest/kotlin/uk/gov/justice/digital/hmpps/educationandworkplanapi/app/resource/UpdateInductionTest.kt

```diff
@@ -854,4 +863,8 @@ class UpdateInductionTest : IntegrationTestBase() {
       qualificationsAndTraining = qualificationsAndTraining,
       inPrisonInterests = inPrisonInterests,
     )
+
+  private fun shortDelayForTimestampChecking() {
+    Thread.sleep(200)
```

Comment:
Did you have to bump it from 100 to 200? Due to intermittent failures?
If that is the case I would guess we might need to bump it back to the original 500 so that it always rounds up ?

Reply:
I played around with different figures and settled on 200. I don't recall encountering any errors, however there is a "fundamental flaw" concerning the `isAfter()` tests above, as the timestamp of the child entities will always be milliseconds after the main parent. I need to change this:

```
assertThat(updatedInduction.workExperience!!.modifiedDateTime).isAfter(initialModifiedAt)
```
To this:
```
assertThat(updatedInduction.workExperience!!.modifiedDateTime).isAfter(persistedInduction.workExperience!!.modifiedDateTime)
```
Or perhaps write an extension function 🤔 

However, I'd like to do this in a separate PR so that I can test that the APIs match ASAP.